webapp/src/lib/Ui.ts

```diff
@@ -73,6 +102,121 @@ export interface IDialogConfig {
 export class DialogOKCancelComponent {
     constructor(
         public dialogRef: MatDialogRef<DialogOKCancelComponent>,
-        @Inject(MAT_DIALOG_DATA) public data: IDialogConfig) {
+        @Inject(MAT_DIALOG_DATA) public data: IDialogOKCConfig) {
+    }
+}
+
+export interface IDialogTransactionConfig {
+    title: string;
+    worker: TWorker;
+}
+
+@Component({
+    selector: "app-dialog-transaction",
+    styleUrls: ["./app-dialog-transaction.scss"],
+    templateUrl: "dialog-transaction.html",
+})
+export class DialogTransactionComponent implements OnInit {
```

Comment:
Hum, `UI.ts` is going over 200 lines, maybe extracting a/some/this classes to other files?